Makefile

```diff
@@ -94,7 +94,7 @@ DOCKER_TOOLS = docker run \
   -v $(shell bash -c "mkdir -p /tmp/go-lint-cache; echo /tmp/go-lint-cache"):/root/.cache/golangci-lint \
   -v $$(pwd):/build litd-tools
 
-ITEST_TAGS := integration itest $(LND_RELEASE_TAGS)
+ITEST_TAGS := dev integration itest lowscrypt $(LND_RELEASE_TAGS)
```

Comment:
Hmm I'm not fully sure why the `lowscrypt` buildtag is added here? Maybe the commit message could be extended to specify that as well if this is intended :)

Reply:
Ah, right. Speeds up wallet creation by using weaker encryption. Added that to the commit message.